website/identifiers/clients/datacite.py

```diff
@@ -33,31 +33,58 @@ def build_metadata(self, node):
         """Return the formatted datacite metadata XML as a string.
          """
 
+        contributors = datacite_format_contributors(node.visible_contributors)
+        contributors.append({
+            'nameType': 'Organizational',
+            'contributorType': 'HostingInstitution',
+            'contributorName': 'Open Science Framework',
+            'name': 'Open Science Framework',
+            'nameIdentifiers': [
+                {
+                    'name': 'Open Science Framework',
+                    'nameIdentifier': f'https://ror.org/{settings.OSF_ROR_ID}/',
+                    'nameIdentifierScheme': 'ROR',
+                },
+                {
+                    'name': 'Open Science Framework',
+                    'nameIdentifier': f'https://grid.ac/institutes/{settings.OSF_GRID_ID}/',
+                    'nameIdentifierScheme': 'GRID',
+                }
+            ],
+        })
+
         data = {
-            'identifier': {
-                'identifier': self.build_doi(node),
-                'identifierType': 'DOI',
-            },
+            'identifiers': [
+                {
+                    'identifier': self.build_doi(node),
+                    'identifierType': 'DOI',
+                }
+            ],
             'creators': datacite_format_creators([node.creator]),
-            'contributors': datacite_format_contributors(node.visible_contributors),
+            'contributors': contributors,
```

Comment:
This should be non-bibliographic contribs with bibliographic contribs going in `creators`: https://blog.datacite.org/contributor-information-in-datacite-metadata/